services/src/main/java/org/jboss/windup/web/services/rest/MigrationProjectEndpointImpl.java

```diff
@@ -96,8 +96,14 @@ public void deleteProject(MigrationProject migrationProject)
     }
 
     @Override
+    public void deleteOldProvisionalProjects()
+    {
+        this.migrationProjectService.deleteOldProvisionalProjects(0);
+    }
+
+    @Override
     public Long getProjectIdByName(String title){
-        String jql = "SELECT id FROM MigrationProject p WHERE LOWER(p.title) = LOWER(:title)";
+        String jql = "SELECT id FROM MigrationProject p WHERE LOWER(p.title) = LOWER(:title) AND p.provisional = FALSE";
```

Comment:
Note that if it's 0 minutes, it could lead to race condition as Jess pointed out somewhere, if the user manages to create the project before the deletion happens.